src/services/user.services.ts

```diff
@@ -10,22 +10,22 @@ async function login({
   password: string;
 }): Promise<UserLoginInterface> {
   const response = await axios
-    .post('http://api.safely-app.fr/login', {
+    .post('https://api.safely-app.fr/login', {
       email: username,
       password,
     })
     .then(res => {
       return res.data;
     })
     .catch(err => {
-      return err;
+      throw err;
```

Comment:
Are you catching these exceptions somewhere ? I found it for the logout but not for the login and register functions.

Reply:
Yes, I am catching these exceptions when calling the api calls from the services in actions :

![image](https://user-images.githubusercontent.com/62654884/120085510-1b192100-c0d9-11eb-8337-643502015ba1.png)

If the API call went wrong, I will catch it and dispatch an action failure which will be storing an error message in my state. Then in my component I'll be checking if anything went wrong after calling the api like that :

![image](https://user-images.githubusercontent.com/62654884/120085540-4e5bb000-c0d9-11eb-9c67-7400c4f7fec9.png)

I am parsing the array 'error' which is the state where I'll be storing error messages if the fetching went wrong, if there is something wrong my component will do something and then I'll be resetting my state containing error messages for future calls